src/components/ObsEdit/Sheets/GeoprivacySheet.js

```diff
@@ -0,0 +1,80 @@
+// @flow
+
+import CheckBox from "@react-native-community/checkbox";
```

Comment:
This looks nice, but it's off spec and I don't think it's necessary given https://callstack.github.io/react-native-paper/docs/components/RadioButton/ (which is also closer to the designs). Did you try the RN paper version?